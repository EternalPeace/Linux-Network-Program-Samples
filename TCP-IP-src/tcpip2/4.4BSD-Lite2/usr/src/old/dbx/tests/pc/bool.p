program booltest (input, output);
var b : boolean;
begin
    b := true;
    b := false;
end.
