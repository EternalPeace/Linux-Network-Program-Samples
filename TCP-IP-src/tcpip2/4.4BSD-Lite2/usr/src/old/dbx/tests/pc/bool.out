reading symbolic information ...
[1] stop at 4
[1] stopped in program at line 4
    4       b := true;
false 
stopped in program at line 5
    5       b := false;
true 
false 
true 

